### PR TITLE
Fix bootstrap version in SelectPath component

### PR DESCRIPTION
**Pull request by anluh**

### CODE REVIEW

File: openy_af4_vue_app/src/components/steps/SelectPath.vue [hunk 1]
```diff
@@ -72,13 +72,22 @@ export default {
...
     }
   },
   data() {
     return {
       image: this.backgroundImage.mobile
     }
   },
+  computed: {
+    pathClasses() {
+      return this.bsVersion === 4 ? 'col-6 col-md-3' : 'col-xs-6 col-md-3'
```
Comment (duozersk) [inline openy_af4_vue_app/src/components/steps/SelectPath.vue]: It was and should be `col-xs-6 col-sm-3` for BS3.

**Review by duozersk — CHANGES_REQUESTED** [review]

File: openy_af4_vue_app/src/components/steps/SelectPath.vue (hunk 1 above)
Comment (anluh) [reply openy_af4_vue_app/src/components/steps/SelectPath.vue]: Done in new commit

**Review by duozersk — APPROVED** [review]
Good job!

**Comment by duozersk** [comment]
NOTE: Requires artifacts rebuild.

**Review by podarok — APPROVED** [review]

**Pull request merged**